install.lua

```diff
@@ -12,6 +12,11 @@ local function formatFS()
       end
     end
     fs.delete("AxiomUI")
+    print("Delete extra files?")
+    if read() == "y" then
+      fs.delete("install.lua")
```

Comment:
Wouldn't it be better to remove the current running program, as long as it isn't pastebin?

Reply:
No. It gets written by gitget. So even if we did not download it, it will exist

Reply:
You're right.